src/styles/blocks/stars.scss

```
@@ -0,0 +1,30 @@
+.card__stars {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  font-weight: 400;
+  font-size: 10px;
+  margin-top: 16px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

```suggestion
&__star {
```
Attach this selector to `.stars`